ingestion/src/metadata/ingestion/source/dashboard/metabase/metadata.py

```diff
@@ -108,7 +109,9 @@ def get_dashboard_name(self, dashboard: MetabaseDashboard) -> str:
         """
         return dashboard.name
 
-    def get_dashboard_details(self, dashboard: MetabaseDashboard) -> dict:
+    def get_dashboard_details(
+        self, dashboard: MetabaseDashboard
+    ) -> MetabaseDashboardDetails:
```

Comment:
This should be `Optional[MetabaseDashboardDetails]` as per the `client.get_dashboard_details` method.

Reply:
will do the change